distro/configs/odoo/initializer_config/custom_fields/ir.model.fields.2000.csv

```diff
@@ -0,0 +1,3 @@
+"id","name","field_description","model_id","ttype","index","store","readonly","relation","size","related"

```

Comment:
`ttype` looks like a typo?

Reply:
I have mostly tried to replicate what was already present [here](https://github.com/mekomsolutions/odoo-config-haiti/blob/master/configuration/custom_fields/ir.model.fields.2000.csv?plain=1#L1)